src/displayapp/screens/SystemInfo.cpp

```diff
@@ -161,7 +161,7 @@ std::unique_ptr<Screen> SystemInfo::CreateScreen2() {
                         brightnessController.ToString(),
                         resetReason);
   lv_obj_align(label, lv_scr_act(), LV_ALIGN_CENTER, 0, 0);
-  return std::unique_ptr<Screen>(new Screens::Label(1, 4, app, label));
+  return std::unique_ptr<Screen>(new Screens::Label(1, 5, app, label));
```

Comment:
It's probably worth refactoring this constant out into a separate constant since it's repeated in several places. That would make an issue like this less likely to occur in the future.